### DISCUSSION TITLE
I like the idea of floating label from angular material. How do i do it in daisyui?

### DISCUSSION THREAD

**Discussion by mahesh-riddles** [saadeghi/daisyui#2601] [Q&A] [answered]
Is it okay to mix angular material as well as daisy ui?
I have most of the components in daisy ui. Just missing the floating label.
It looks nicer, Has anyone implemented floating label in daisyui?

**Comment by saadeghi** [comment, marked as answer]
Yes, see this: https://github.com/saadeghi/daisyui/issues/298#issuecomment-967194458

**Reply by mahesh-riddles** [reply]
That is wonderful. I am really proud that your respect the copyright of others. 
So nice.
